styleguide/source/assets/scss/00-base/_typography.scss

```diff
@@ -44,6 +44,12 @@ h5,
   @include font-size($h5-font-sizes);
 }
 
+p {
```

Comment:
Should these styles be on the `body`?

Reply:
body already has a style but the p tag had different sizing. This is to ensure all p are the same size no matter what.

Reply:
I agree that all `<p>` should be the same I guess my question is why is the body not set to the default `<p>` sizing?

Reply:
If we need to change the size of the body copy of page that needs to happen on the default html element. All sizing of everything else should be relative to the base body copy. This kind of deviation is going to be a nightmare for maintaining the code moving forward.

Reply:
agreed making the appropriate changes now and removing the a tag inside of p.